www/js/app.js

```diff
@@ -785,8 +785,10 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                             if (protocol === 'file:') {
                                 message += "\n\nYou seem to be opening kiwix-js with the file:// protocol. You should open it through a web server : either through a local one (http://localhost/...) or through a remote one (but you need SSL : https://webserver/...)";
                             }
-                            alert(message);                        
-                            setContentInjectionMode("jquery");
+                            uiUtil.systemAlert("Alert", message, false)
+                                .then(function () {
```

Comment:
Just a style point: either put the `.then(...{` at the end of the previous line, or else on the line below without further indenting (Promises are supposed to avoid the "callback from hell" indenting problem, so usually a flat style is used.) If in doubt, as the previous line is short, I'd just move it up there and then remove one level of indenting from the function.